tools/common/Driver.cs

```diff
@@ -384,7 +384,7 @@ static void SetCurrentLanguage ()
 				var culture = CultureInfo.GetCultureInfo (lang);
 				if (culture != null) {
 					CultureInfo.DefaultThreadCurrentCulture = culture;
-					ErrorHelper.Warning (123, $"The current language was set to '{culture.DisplayName}' according to the LANG environment variable (LANG={lang_variable}).");
```

Comment:
I don't think this has been released (need to check 15.4) but, if not, then the error doc should be updated

Reply:
@spouliot 15.4 does not have MT0123, so I removed the warning from the error doc.